aten/src/THC/generic/THCTensorMasked.cu

```diff
@@ -126,7 +126,7 @@ void THCTensor_(maskedCopyBool)(THCState* state,
              "mask and tensor must have the same number of elements");
 
   // Determine our output size
-  ptrdiff_t totalElements = THTensor_wrap(mask).sum().item<ptrdiff_t>();
+  ptrdiff_t totalElements = THTensor_wrap(mask).sum().item<int64_t>();
```

Comment:
Why return type should be `ptrdiff_t` rather than `int64_t`?
```suggestion
  int64_t totalElements = THTensor_wrap(mask).sum().item<int64_t>();
```